### PR TITLE
enhancement: use more semantic methods

### PR DESCRIPTION
**Pull request by kezhenxu94**
Motivation:

Improve code readability

Modification:

Replace some bitwise operations with more semantic methods

### CODE REVIEW

**Comment by normanmaurer** [comment]
I am not sure I agree with this... Let me think a bit about it

File: transport/src/main/java/io/netty/channel/nio/NioHandler.java [hunk 1]
```diff
@@ -617,14 +617,14 @@ private void processSelectedKey(SelectionKey k, AbstractNioChannel ch) {
...
-            if ((readyOps & SelectionKey.OP_WRITE) != 0) {
+            if (k.isWritable()) {
                 // Call forceFlush which will also take care of clear the OP_WRITE once there is nothing left to write
                 ch.unsafe().forceFlush();
             }
 
             // Also check for readOps of 0 to workaround possible JDK bug which may otherwise lead
             // to a spin loop
-            if ((readyOps & (SelectionKey.OP_READ | SelectionKey.OP_ACCEPT)) != 0 || readyOps == 0) {
+            if (readyOps == 0 || k.isReadable() || k.isAcceptable()) {
```
Comment (normanmaurer) [inline transport/src/main/java/io/netty/channel/nio/NioHandler.java]: this will be a bit slower as you do one extra check compared to bitwise stuff

Comment (normanmaurer) [reply transport/src/main/java/io/netty/channel/nio/NioHandler.java]: @kezhenxu94 basically what I am saying that before you needed 2 checks while with your its 3 now. That said I think it really not matters too much.

**Comment by normanmaurer** [comment]
@kezhenxu94 I think I actually would prefer to just keep it as it is as we then use bitwise consistent in the implementation and not mixing stuff

**Comment by kezhenxu94** [comment]
Thanks for reviewing

**Pull request closed without merging**